.github/workflows/ci.yml

```
@@ -128,6 +128,155 @@ jobs:
+        with:
+          node-version: '20'
+
+      - name: Install root dependencies
+        run: npm ci
+
+      - name: Bundle canonical OpenAPI artifacts
+        run: |
+          bash scripts/contracts/bundle.sh
+
+      - name: Start Prism mocks (revenue + health) and run usability smoke
+        run: |
```

actionlint (1.7.10)

[error] shellcheck reported issue in this script: SC2034:warning:12:1: i appears unused. Verify use (or export if used externally)

(shellcheck)